### PR TITLE
No Jira: some release script tweaks

### PR DESCRIPTION
**Pull request by stillalex**
https://issues.apache.org/jira/browse/SOLR-XXXXX

# Description

Some tweaks based on working on the Solr release.

I ended up creating a new build page for Solr, looking for some feedback on this (including if this is needed or not) https://cwiki.apache.org/confluence/display/SOLR/JenkinsReleaseBuilds+-+Solr

# Solution

Please provide a short description of the approach taken to implement your solution.

# Tests

Please describe the tests you've developed or run to confirm this patch implements the feature or solves the problem.

# Checklist

Please review the following and check all that apply:

- [ ] I have reviewed the guidelines for [How to Contribute](https://wiki.apache.org/solr/HowToContribute) and my code conforms to the standards described there to the best of my ability.
- [ ] I have created a Jira issue and added the issue ID to my pull request title.
- [ ] I have given Solr maintainers [access](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to contribute to my PR branch. (optional but recommended)
- [ ] I have developed this patch against the `main` branch.
- [ ] I have run `./gradlew check`.
- [ ] I have added tests for my changes.
- [ ] I have added documentation for the [Reference Guide](https://github.com/apache/solr/tree/main/solr/solr-ref-guide)

### CODE REVIEW

**Review by risdenk — APPROVED** [review]

File: dev-tools/scripts/releaseWizard.yaml [hunk 1]
```diff
@@ -636,7 +636,7 @@ groups:
       Ask on dev@ for input. Ideally the timing of this request mostly coincides with the
       release branch creation. It's a good idea to remind the devs of this later in the release too.
 
-      NOTE: Do not add every single JIRA issue, but distill the Release note into important changes!
+      NOTE: Do not add every single JIRA issue, but distill the Release notes into important changes!
```
Comment (madrob) [inline dev-tools/scripts/releaseWizard.yaml:639]: capitalization: I think this should either be `release notes` or `Release Notes` (not sure how it is referred to elsewhere, most important is consistency.)

Comment (stillalex) [reply dev-tools/scripts/releaseWizard.yaml]: agreed, used 'release notes' so it's same as above.

**Comment by stillalex** [comment]
thanks for the reviews @risdenk and @madrob!
could you also take a quick look at https://cwiki.apache.org/confluence/display/SOLR/JenkinsReleaseBuilds+-+Solr and suggest changes if any.

**Comment by stillalex** [comment]
in an interesting turn of events I managed to transition to RC2 without wiping the svn state clean by clicking 'no' on the prompt... I am wondering if this is the right way to go about it.

```
  ┌─────────────────────────────────────────────────────────────────────────┐
  │                                                                         │
  │  Solr ReleaseWizard                                                     │
  │                                                                         │
  │  Releasing Solr 9.4.0 RC1                                               │
  │                                                                         │
  │                                                                         │
  │  Welcome to the release wizard. From here you can manage the process    │
  │  including creating new RCs. All changes are persisted, so you can      │
  │  exit any time and continue later. Make sure to read the Help section.  │
  │                                                                         │
  │                                                                         │
  │    1 - Go to checklist (RC1)                                            │
  │    2 - Abort RC1 and start a new RC2                                    │
  │    3 - Clear and restart current RC                                     │
  │    4 - Clear all state, restart the 9.4.0 release                       │
  │    5 - Start release for a different version                            │
  │    6 - Generate Asciidoc guide for this release                         │
  │    7 - Help                                                             │
  │    8 - Exit                                                             │
  │                                                                         │
  │                                                                         │
  │  ® 2022 The Solr project. Licensed under the Apache License 2.0         │
  │  Script version v9.5.0)                                                 │
  │                                                                         │
  │                                                                         │
  └─────────────────────────────────────────────────────────────────────────┘
  >> 2

Q: Are you sure? This will abort current RC (y/n): y


import_svn done
Looks like you uploaded artifacts for ee474b7db483c2242ce1d75074258236ca22103b to svn which needs to be removed.

  cd ~/.solr-releases/9.4.0/solr
  svn -m "Remove cancelled Solr 9.4.0 RC1" rm https://dist.apache.org/repos/dist/dev/solr/solr-9.4.0-RC1-rev-ee474b7db483c2242ce1d75074258236ca22103b

You will not be prompted for each command but will see the output of each. If one command fails the execution will stop.

Q: Do you want me to run these commands now? (y/n): n


Saving
Started RC2

  ┌─────────────────────────────────────────────────────────────────────────┐
  │                                                                         │
  │  Solr ReleaseWizard                                                     │
  │                                                                         │
  │  Releasing Solr 9.4.0 RC2                                               │
  │                                                                         │
  │                                                                         │
  │  Welcome to the release wizard. From here you can manage the process    │
  │  including creating new RCs. All changes are persisted, so you can      │
  │  exit any time and continue later. Make sure to read the Help section.  │
  │                                                                         │
  │                                                                         │
  │    1 - Go to checklist (RC2)                                            │
  │    2 - Abort RC2 and start a new RC3                                    │
  │    3 - Clear and restart current RC                                     │
  │    4 - Clear all state, restart the 9.4.0 release                       │
  │    5 - Start release for a different version                            │
  │    6 - Generate Asciidoc guide for this release                         │
  │    7 - Help                                                             │
  │    8 - Exit                                                             │
  │                                                                         │
  │                                                                         │
  │  ® 2022 The Solr project. Licensed under the Apache License 2.0         │
  │  Script version v9.5.0)                                                 │
  │                                                                         │
  │                                                                         │
  └─────────────────────────────────────────────────────────────────────────┘
```

**Comment by HoustonPutman** [comment]
I don't see anything wrong here... I mean you should delete the rc1 artifacts, not sure why you said `n` there, but they can be deleted manually afterwards. Is there a reason you wanted to keep the artifacts/state?

**Comment by stillalex** [comment]
> I don't see anything wrong here... I mean you should delete the rc1 artifacts, not sure why you said n there, but they can be deleted manually afterwards. Is there a reason you wanted to keep the artifacts/state?

I agree. there was some confusion in the moment. I wanted to see what the steps are, so I clicked 'Abort' expecting to see a sequence of steps where I can say 'n' and return. it asked 'are you sure' and I said 'yes', but then said 'no' to svn removal step, not expecting that the abort happened already. so 90% my own confusion based on learning a pattern of 'click into some option, being shown something like - this are the commands yes/no'. I was able to copy/paste the commands later and successfully cleanup the dist repo.

**Comment by janhoy** [comment]
A tip if you just want to familiarize yourself with the release process is to generate html version of wizard on first page. Or use dry-run mode.

**Comment by stillalex** [comment]
more troubles with release steps `Add the new version to the Apache Release Reporter`

```Could not save. Make sure you have filled out all fields and have access to this committee data! For further inquiries, please contact dev@community.apache.org
Error: User stillalex not a member of PMC solr nor an ASF member ```

I think we can update the wizard to specify you can't do this unless PMC member

**Comment by stillalex** [comment]
Step `7 - Stop promoting old releases `

```
*9.3.0*
Run these commands to delete proposed versions from distribution directory.
Note, as long as we have some releases (7.x, 8.x) in Lucene dist repo and other
releases (9.0 ->) in the Solr dist repo, we may need to delete two places.

WARNING: Validate that the proposal is correct!

  cd ~/.solr-releases/9.4.0/solr
  # Delete from Lucene dist area
  svn rm -m "Stop publishing old Solr releases" \
      https://dist.apache.org/repos/dist/release/lucene/solr/9.3.0
  # Delete from new Solr dist area
  svn rm -m "Stop publishing old Solr releases" \
      https://dist.apache.org/repos/dist/release/solr/solr/9.3.0

You will get prompted before running each individual command.

Q: Do you want me to run these commands now? (y/n): y


# Delete from Lucene dist area


Q: Shall I run 'svn rm -m "Stop publishing old Solr releases"  https://dist.apache.org/repos/dist/release/lucene/solr/9.3.0' in folder '/Users/alexdeparvu/.solr-releases/9.4.0/solr' (y/n): y


Wait until command completes... Full log in /Users/alexdeparvu/.solr-releases/9.4.0/RC2/logs/stop_promoting_old/01_svn-rm-solr.log

# Delete from Lucene dist area

                                                                                
svn: E160013: URL 'https://dist.apache.org/repos/dist/release/lucene/solr/9.3.0' does not exist

WARN: Command svn rm -m "Stop publishing old Solr releases"  https://dist.apache.org/repos/dist/release/lucene/solr/9.3.0 returned with error
WARNING: One or more commands failed, you may want to check the logs
```

this fails to delete old Solr release because Lucene entry is not there.

**Pull request merged**